extensions/emmet/src/removeTag.ts

```diff
@@ -51,16 +51,40 @@ function getRangeToRemove(editor: vscode.TextEditor, rootNode: HtmlNode, selecti
 
 	let ranges = [openRange];
 	if (closeRange) {
-		for (let i = openRange.start.line + 1; i <= closeRange.start.line; i++) {
+		const indentAdjustAmount: number = getAdjustedIndentAmount(editor, openRange, indentInSpaces);
+		for (let i = openRange.start.line + 1; i <= closeRange.start.line && indentAdjustAmount !== 0; i++) {
 			let lineContent = editor.document.lineAt(i).text;
-			if (lineContent.startsWith('\t')) {
-				ranges.push(new vscode.Range(i, 0, i, 1));
-			} else if (lineContent.startsWith(indentInSpaces)) {
-				ranges.push(new vscode.Range(i, 0, i, indentInSpaces.length));
+			if (lineContent.startsWith('\t'.repeat(indentAdjustAmount))) {
+				ranges.push(new vscode.Range(i, 0, i, indentAdjustAmount));
+			} else if (lineContent.startsWith(indentInSpaces.repeat(indentAdjustAmount))) {
+				ranges.push(new vscode.Range(i, 0, i, indentInSpaces.length*indentAdjustAmount));
 			}
 		}
 		ranges.push(closeRange);
 	}
 	return ranges;
 }
 
+function getAdjustedIndentAmount(editor: vscode.TextEditor ,openRange: vscode.Range, indentInSpaces: string): number {
+	const startLineContent: string = editor.document.lineAt(openRange.end.line).text;
+	// If there is some content in the same line as opening tag, then don't adjust indentation.
+	if(startLineContent.length !== openRange.end.character) {
```

Comment:
Tiny nit pick: The code base has a space between the if and open brace.

`if (`